### PR TITLE
Include partition in default scope feature info added in the Flex doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add info about the new include partition in default partition feature in the Create/Edit/View flex doc.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Update documentation page for both default scope and data model column renaming story](https://sumologic.atlassian.net/browse/DOCS-126)

### CODE REVIEW

File: docs/manage/partitions/flex/create-edit-partitions-flex.md [hunk 1]
```diff
@@ -66,7 +66,8 @@ You can make some changes to an existing partition:  
   By default, Sumo Logic internal partitions like `sumologic_audit_events`, `sumologic_volume`, and so on, have the same retention period as the Default Continuous Index. You can change the retention period for any of these internal partitions as desired.
   :::
 * You can change the data forwarding configuration.
-* You cannot change the name of partition, routing expression, or reuse a partition name.  
+* You cannot change the name of partition, routing expression, or reuse a partition name.
+* You cannot edit the audit index partition to include in the default scope.
```
Comment (jpipkin1) [inline docs/manage/partitions/flex/create-edit-partitions-flex.md:70]:
```suggestion
* You cannot change the name of a partition, the routing expression, or reuse a partition name.
* You cannot edit the audit index partition to include it in the default scope. 
```

**Review by jpipkin1 — APPROVED** [review]

File: docs/manage/partitions/flex/create-edit-partitions-flex.md (hunk 1 above)
Comment (jpipkin1) [reply docs/manage/partitions/flex/create-edit-partitions-flex.md:70]: @kimsauce, I'm confused by this statement: "You cannot change the name of partition, routing expression, or reuse a partition name." A little further down, the dialog to edit a partition seems to show that you can change the name of a partition and its routing expression.

Comment (JV0812) [reply docs/manage/partitions/flex/create-edit-partitions-flex.md:70]: @jpipkin1 we cannot edit/change the partition name and routing expression once it is created. Even if you check the next steps on **How to edit a partition** you can just see we only allow editing of retention period and modifying the configuration of data forwarding.

Comment (jpipkin1) [reply docs/manage/partitions/flex/create-edit-partitions-flex.md:70]: Thanks for the explanation, @JV0812.

File: docs/manage/partitions/flex/create-edit-partitions-flex.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ To create or edit a Partition, you must be an account Administrator or have th
 1. Click **+ Add Partition**.
 1. The **Create New Partition** pane appears.<br/><img src={useBaseUrl('img/partitions-data-tiers/create-new-partition-flex.png')} alt="create-new-partition-flex.png"  style={{border:'1px solid gray'}} width="300"/>
 1. **Name**. Enter a name for the Partition. Partitions must be named alphanumerically, with no special characters, with the exception of underscores (`_`). However, a Partition name cannot start with `sumologic_` or an underscore `_`.
+1. (Optional) **Include this partition in default scope**. This checkbox is already selected. Deselect this checkbox if you need to exclude this partition as the default scope in your search.
```
Comment (krastogi-eng) [inline docs/manage/partitions/flex/create-edit-partitions-flex.md:28]: Along with this we should explain to the customers what does default scope mean as it is a new concept that we are introducing to them

**Pull request merged**